ocaml/xapi/import.ml

```diff
@@ -432,6 +432,14 @@ module VM : HandlerTools = struct
           "0"|"1"|"2"|"3" as order -> { vm_record with API.vM_ha_restart_priority = "restart"; API.vM_order = Int64.of_string (order) }
         | _ -> vm_record;
       in
+      (* Initialize platform["device-model"] if it is not set *)
```

Comment:
Is this code properly indented? Could you run ocp-indent over it?

Reply:
Hi @lindig,
I have used `ocp-indent ocaml/xapi/import.ml`, this is the formatted code.